app/src/sharedTest/java/org/oppia/android/app/parser/HtmlParserTest.kt

```diff
@@ -412,20 +439,29 @@ class HtmlParserTest {
     assertThat(textView.textDirection).isEqualTo(View.TEXT_DIRECTION_LTR)
   }
 
+  // TODO(#3840): Make this test work on Espresso & Robolectric.
```

Comment:
Should this just be Espresso? It seems to already be enabled for Robolectric.

Reply:
Done.